src/MovieList.jsx

```diff
@@ -63,7 +64,7 @@ function MovieList({
   const Cards = data?.results?.slice(0, AMOUNT_MOVIES).map((movie, index) => (
     <MovieCard
       key={movie.id}
-      movieImage={`https://image.tmdb.org/t/p/w342${movie?.poster_path}`}
+      movieImage={`https://image.tmdb.org/t/p/w780${movie?.poster_path}`}
```

Comment:
What's the difference between w342 and w780?

Reply:
The resolution of the image